scaladoc/src/dotty/tools/scaladoc/ScaladocSettings.scala

```diff
@@ -120,5 +120,8 @@ class ScaladocSettings extends SettingGroup with AllScalaSettings:
   val generateInkuire: Setting[Boolean] =
     BooleanSetting("-Ygenerate-inkuire", "Generates InkuireDB and enables Hoogle-like searches", false)
 
+  val legacyAPILayout: Setting[Boolean] =
+    BooleanSetting("-Ylegacy-api-layout", "Keep all api member inside `api` directory", false)
```

Comment:
I’m wondering if we could come up with a more self-descriptive name, maybe `-Yapi-subdirectory`?